app/tests/src/test_ingest_bem_pdfs.py

```diff
@@ -55,3 +83,30 @@ def test__ingest_policy_pdfs(caplog, app_config, db_session, policy_s3_file, fil
         # assert "Temporary Assistance to Needy Families" not in second_chunk.content
         # assert "The Food Assistance Program" in second_chunk.content
         # assert math.isclose(second_chunk.mpnet_embedding[0], -0.82242084, rel_tol=1e-3)
+
+
+def test__enrich_text():
+    with smart_open("/app/tests/docs/100.pdf", "rb") as file:
+        enriched_text_list = _enrich_texts(file)
+
+        assert len(enriched_text_list) == 9
+        enriched_text_item = enriched_text_list[0]
+        assert isinstance(enriched_text_item, EnrichedText)
```

Comment:
Add assertions to check that `EnrichedText` items have the correct headings, type, and page_number associated with them.